featurebyte/feast/patch.py

```diff
@@ -0,0 +1,92 @@
+"""
+This module functions used to patch the Feast library.
+"""
+from __future__ import annotations
+
+from typing import List
+
+from collections import defaultdict
+
+# pylint: disable=no-name-in-module
+from feast import OnDemandFeatureView
+from feast.online_response import OnlineResponse
+from feast.protos.feast.serving.ServingService_pb2 import FieldStatus, GetOnlineFeaturesResponse
+from feast.type_map import python_values_to_proto_values
+from google.protobuf.timestamp_pb2 import Timestamp  # type: ignore[import]
+
+
+def augment_response_with_on_demand_transforms(
+    online_features_response: GetOnlineFeaturesResponse,
+    feature_refs: List[str],
+    requested_on_demand_feature_views: List[OnDemandFeatureView],
+    full_feature_names: bool,
+) -> None:
+    """Computes on demand feature values and adds them to the result rows.
```

Comment:
Would be good to add a note in the docstring on the main changes compared to the original version.

Reply:
Currently the comment are added at line 65, 73 & 74. Will update the comment here in the following PR.